tasks/new_e2e_tests.py

```diff
@@ -124,7 +124,7 @@ def run(
         parsed_params[parts[0]] = parts[1]
 
     if local_package:
-        parsed_params["ddagent:localPackage"] = local_package
+        parsed_params["ddagent:localPackage"] = Path(local_package).absolute().as_posix
```

Comment:
```suggestion
        parsed_params["ddagent:localPackage"] = Path(local_package).absolute().as_posix()
```
Isn't it a function?